src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -679,28 +670,25 @@ public void testExistingDirectoryWithViolations() throws Exception {
         final String msgKey = "maxLen.file";
         final String bundle = "com.puppycrawl.tools.checkstyle.checks.sizes.messages";
 
-        exit.checkAssertionAfterwards(() -> {
-            final String expectedPath = getFilePath("") + File.separator;
-            final StringBuilder sb = new StringBuilder(28);
-            sb.append(auditStartMessage.getMessage())
-                    .append(EOL);
-            final String format = "[WARN] " + expectedPath + outputValues[0][0] + ".java:"
-                    + outputValues[0][1] + ": ";
-            for (String[] outputValue : outputValues) {
-                final String localizedMessage = new LocalizedMessage(1, bundle,
-                        msgKey, new Integer[] {Integer.valueOf(outputValue[2]), allowedLength},
-                        null, getClass(), null).getMessage();
-                final String line = format + localizedMessage + " [FileLength]";
-                sb.append(line).append(EOL);
-            }
-            sb.append(auditFinishMessage.getMessage())
-                    .append(EOL);
-            assertEquals("Unexpected output log", sb.toString(), systemOut.getLog());
-            assertEquals("Unexpected system error log", "", systemErr.getLog());
-        });
-
         Main.main("-c", getPath("InputMainConfig-filelength.xml"),
                 getPath(""));
+        final String expectedPath = getFilePath("") + File.separator;
+        final StringBuilder sb = new StringBuilder(28);
```

Comment:
`StringBuilder(28);`
just wondering why it is like this...

Reply:
git blame says it comes from #4491. Well, any big enough value is good here.